### ISSUE TITLE
App working but getting javascript error on starting 

### ISSUE THREAD

**Issue by DjordjePetrovic** [chuyik/electron-boilerplate#1]
Hey man! Thx for simple and clean boilerplate! Just wanted to aks you, I downloaded your code and tried to build it just to see how it works and I manage to open the app but I get https://dl.dropboxusercontent.com/s/nfk5zvyp4tbxdtt/Screen%20Shot%202016-01-06%20at%201.52.58%20PM.png?dl=0 any idea? Thanks!

**Comment by DjordjePetrovic** [comment]
@chuyik Thanks!

**Comment by chuyik** [comment]
Hi @DjordjePetrovic good to see you use it, and I reproduced this issue, sorry for that.

The reason is the `node-notifier` will fail to load if the app is packaged within asar.
I've used the latest `electron-packager` to ignore some binary files of `node-notifier` any no errors prompted again.

TLDR:
1. use electron-packager 5.2.0
2. use `asar-unpack-dir` like commit 360cd1
3. happy again!
